docs/reference/go-gno-compatibility.md

```diff
@@ -297,31 +297,31 @@ Legend:
-| go list           |                           |                                                                       |
-| go mod            | gno mod                   |                                                                       |
-| + go mod init     | gno mod init              | same behavior                                                         |
-| + go mod download | gno mod download          | same behavior                                                         |
-| + go mod tidy     | gno mod tidy              | same behavior                                                         |
-| + go mod why      | gno mod why               | same intention                                                        |
-|                   | gno transpile             |                                                                       |
-| go work           |                           |                                                                       |
-|                   | gno repl                  |                                                                       |
-| go run            | gno run                   |                                                                       |
-| go test           | gno test                  | limited compatibility                                                 |
-| go tool           |                           |                                                                       |
-| go version        |                           |                                                                       |
-| go vet            |                           |                                                                       |
-| golint            | gno lint                  | same intention                                                        |
+| go command        | gno command                  | comment                                                               |
+|-------------------|------------------------------|-----------------------------------------------------------------------|
+| go bug            | gno bug                      | same behavior                                                         |
+| go build          | gno tool transpile -gobuild  | same intention, limited compatibility                                 |
+| go clean          | gno clean                    | same intention, limited compatibility                                 |
+| go doc            | gno doc                      | limited compatibility; see https://github.com/gnolang/gno/issues/522  |
+| go env            | gno env                      |                                                                       |
+| go fix            |                              |                                                                       |
+| go fmt            | gno fmt                      | gofmt (& similar tools, like gofumpt) works on gno code.              |
```

Comment:
I think it's important to note that it includes `import` by default compared to the original `go fmt`, so it will try to format and resolve dependencies. (It should not fail)